tasks/main.yml

```diff
@@ -45,23 +45,25 @@
     command: "{{ maintain_command }} packages lock"
     when: maintain_command is defined
   when: install_receptor_from_rpm|bool
-- name: Extract account information from Satellite and write configs
+- name: Retrieve organization details
   block:
-  - name: List all organizations in Satellite
-    uri:
-      url: "{{ satellite_url }}/katello/api/organizations"
-      user: "{{ satellite_user }}"
-      password: "{{ satellite_password }}"
-      return_content: yes
-      validate_certs: "{{ satellite_validate_certs }}"
-      force_basic_auth: yes
-    register: sat_orgs_json
-  - name: Extract organization configuration per account and save to disk
-    include_tasks: extract_orgs.yml
-    loop: "{{ sat_orgs_json.json | json_query('results[*].id') }}"
-    when: not org_id in skip_satellite_org_id_list
-    loop_control:
-      loop_var: org_id
+    - name: Dump organization details
```

Comment:
what permissions this sets for the file? could someone read the content? (race condition)

Reply:
644, owned by foreman:foreman

Reply:
A temporary file is now used. It is still owned by `foreman:foreman`, but has 600 permissions